code/client/client.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <yarp/os/all.h>
+
+using namespace std;
+using namespace yarp::os;
+
+int main(int argc, char * argv[])
+{
+	if(argc<2){
+		cout << "Please supply a port name for client" << endl;
+		return 1;
+	}
+
+	const char *charName = argv[1];
+
+    Network yarp;	//initialize yarp
+    Semaphore sem;
+    Port portP;
+    portP.open(charName);
+    
+	int intN = 0, intTh = 0;
+
```

Comment:
Unless you specifically require an editor to display special characters, `tabs` are hidden and formatted differently depending on the platforms. That's the reason why we have to use always `spaces` instead `tabs`. Normally, an editor has the option to replace `tab` with e.g. `4 spaces` when the user presses the tab key.

So, check out your editor's settings.